### ISSUE TITLE
Decoder outputs blank lines for translation

### ISSUE THREAD

**Issue by minump** [tensorflow/tensor2tensor#1639]
### Description

Using transformer model with transformer_base_single_gpu for En-De machine translation. For certain inputs, the decoder output is <EOS> <PAD> <PAD> .. .. (essentially a blank line) for certain translations. Tried to check the softmax output at the decoder end for the probabilities. I am unable to find the code for the same. Any help is appreciated.

### Environment information

```
OS: Linux

$ pip freeze | grep tensor
mesh-tensorflow==0.0.5
tensor2tensor==1.13.4
tensorboard==1.14.0
tensorflow==1.13.1
tensorflow-datasets==1.0.2
tensorflow-estimator==1.14.0
tensorflow-gpu==1.14.0
tensorflow-metadata==0.14.0
tensorflow-probability==0.7.0

$ python -V
Python 3.7.3
```

### For bugs: reproduction and error logs
Blank line as inference output.
in log_decode_results
inputs=  [[  862]
 [   34]
 [    9]
 [  540]
 [ 4802]
 [   36]
 [  939]
 [  570]
 [13646]
 [ 3808]
 [   15]
 [14945]
 [ 2773]
 [   10]
 [ 1583]
 [  180]
 [13646]
 [21722]
 [ 8219]
 [13332]
 [   41]
 [11789]
 [  101]
 [    1]]
outputs=  [1 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0]
```
# Steps to reproduce:
t2t-trainer --data_dir=$DATA_DIR --output_dir=$OUTPUT_DIR --problem=translate_ende_wmt32k --model=transformer --hparams_set=transformer_big_single_gpu --hparams="batch_size=4096,max_length=150"

t2t-decoder --data_dir=$DATA_DIR --output_dir=$OUTPUT_DIR --problem=translate_ende_wmt32k --model=transformer --hparams_set=transformer_big_single_gpu --hparams="batch_size=4096, max_length=150" --decode_hparams="beam_size=4,alpha=0.6" --decode_from_file=$FILE_TO_DECODE --decode_to_file=$DECODED_FILE
```

```
# Error logs:
...
```
